.eslintrc

```diff
@@ -159,7 +159,9 @@
           "xpi",
           "xyz",
           "yaml",
-          "cpu"
+          "cpu",
+          "Autocompletion",
+          "Live Autocompletion"
```

Comment:
No need to add "Live Autocompletion" as live is a word and Autocompletion is defined above